src/status_im2/common/scroll_page/view.cljs

```diff
@@ -117,7 +117,7 @@
       [:<>
        [:f> scroll-page-header @scroll-height height name page-nav-right-section-buttons
         logo sticky-header top-nav title-colum navigate-back?]
-       [rn/scroll-view
+       [rn/flat-list
```

Comment:
@flexsurfer i preferred flat-list here because communities may end up being quite a long list which may have cause perfomance issues with scroll-view

Reply:
but i don't see a data, only header is used, could you elaborate?

Reply:
Before this was using a scroll-view and the data was a map-indexed collection which would probably have perfomance issue if the collection became quite large. The header renders a flat-list populated with communities and in this context communities of the discover screen.  Using a scroll-view in this case leads to having the warning `VirtualizedLists should never be nested inside plain ScrollViews with the same orientation` hence why flat-list is used. The main reason why i used flat-list here is to avoid the warning while using a flat-list to renders discover communities

Reply:
but it doesn't make sense, children must be in the data rendered a list items not in header?

Reply:
@flexsurfer in this case I did not find another way to use a flat-list. The scroll-page component may have to be restructured to be able to render item using render-fn. 

I suggest to merge this as is at the moment and then refactor the scroll-page component so that the data is rendered as list-items as you suggested in a follow up pr. 

The challenge here the scroll-page component is to be used by different screens with the same functionality and the data may largely differ which is why I opted to just render the children in the flat-list header

Reply:
i mean i don't understand why this change is needed then if it doesn't fix anything

Reply:
@flexsurfer maybe its not necessary but my reasoning using a `map-indexed` with scroll-view maybe okay for a small list of communities and messages but when the data is quite large there may be perfomance issue as flat-list is lazy loading while map-indexed is not. I can revert this for now as I look into how to use flat-list in this context if necessary

Reply:
map-indexed with scroll-view is bad and should be never used, but i don't see any changes related to map-indexed i only see that we move content of scroll view into flat list header